qa/tests/features/new/manual/main-table/Presets/Presets.feature

```diff
@@ -1,46 +1,54 @@
-@regression
 Feature: Main table, Presets
   As the Anfisa user I want to apply presets to the variants list on the Main Table page
 
-	Scenario: 01 List of presets
+Background: 
+	Given Main table of "PGP3140_wgs_panel_hl" dataset was opened
+
+Scenario: 01 List of presets
+
+	When  User clicks "Select Filter Preset" drop down menu
+	Then The list of presets should be displayed
 
-		Given Main Table for the "PGP3140_wgs_panel_hl" dataset was opened
-		When  User clicks "Select Filter Preset" drop down menu
-		Then The list of presets should be displayed
+@regression
+@smoke
+Scenario: 02 Apply a preset
 
-	Scenario: 02 Apply a preset
+	Given Presets drop-down menu was opened
+	When User clicks "@SEQaBOO_Hearing_Loss_v_5" preset
+	Then Preset should be applied
```

Comment:
What should a person or a script that checks this scenario do to check that "Preset is applied"?
- Preset name should be written instead of "Select Filter Preset"
- Number of displayed variants in table changes to...
- Number of variants/transcripts in top right corner changes to...
"Preset is applied" doesn't mean any specified check. You can just remove this string